Cesium.CodeGen/Contexts/AssemblyContext.cs

```diff
@@ -165,12 +165,38 @@ public FieldReference GetConstantPoolReference(string stringConstant)
         Debug.Assert(writtenBytes == bufferSize - 1);
 
         var type = GetStubType(bufferSize);
-        field = GenerateFieldForStringConstant(type, data);
-        _stringConstantHolders.Add(stringConstant, field);
+        field = GenerateFieldForDataConstant(type, data);
+        _dataConstantHolders.Add(stringConstant, field);
 
         return field;
     }
 
+    public FieldReference GetConstantPoolReference(byte[] dataConstant)
+    {
+        if (dataConstant.Length >= 10)
+        {
+            // Do not attempt to put large arrays in cache.
+            // Most likely they are single use in the application.
+            // No chance that they would be cached by values.
+            var bufferSize = dataConstant.Length;
+            var type = GetStubType(bufferSize);
+            return GenerateFieldForDataConstant(type, dataConstant);
+        }
+        else
+        {
+            var hash = string.Join("", dataConstant.Select(_ => _.ToString("X2")));
```

Comment:
Come on, let's write a proper `IEqualityComparer` for byte arrays instead of this 😸 

That means the type of `_dataConstantHolders` will have to change, since I prefer to see either a `byte[]` or a byte array wrapper (with cached hash code) as the key.